parsley/message_definitions.py

```diff
@@ -51,6 +51,7 @@
     'DEBUG_MSG':            [BOARD_ID, TIMESTAMP_3, Numeric('level', 4), Numeric('line', 12), ASCII('data', 24)],
     'DEBUG_PRINTF':         [BOARD_ID, ASCII('string', 64)],
     'DEBUG_RADIO_CMD':      [BOARD_ID, ASCII('string', 64)],
+    'ACT_ANALOG_CMD':       [BOARD_ID, TIMESTAMP_3, Enum('actuator', 8, mt.actuator_id), Numeric('act_state', 32)],
```

Comment:
I realize that the Numeric field is set up as a union, but I don't know enough about omnibus to know how you would actually signal to a data sink that a Numeric needs to be interpreted as a float. I would guess that functionality has never been used since afaik no canlib messages prior to the ones being added now have actually used floats.